packages/medusa-source-shopify/package.json

```diff
@@ -47,5 +47,9 @@
     "eslint": "^7.32.0",
     "jest": "^26.6.3"
   },
-  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e"
+  "gitHead": "cd1f5afa5aa8c0b15ea957008ee19f1d695cbd2e",
+  "keywords": [
+    "medusa-plugin",
+    "medusa-plugin-migration"
```

Comment:
suggestion: Should we name these `medusa-plugin-source` instead? Long term, I could imagine these being used for more than just migrations. We've been brainstorming on using these as connectors allowing you to have two-way sync. of products (and potentially more)